src/+dryair/assembleCoefficientMatrices.m

```diff
@@ -20,14 +20,20 @@
     % Alias of SoilVariables
     SV = SoilVariables;
 
+    if ~GroundwaterSettings.GroundwaterCoupling  % Groundwater Coupling is not activated, added by Mostafa
+        indxBotm = 1; % index of bottom layer, by defualt (no groundwater coupling) its layer with index 1, since STEMMUS calcuations starts from bottom to top
+    else % Groundwater Coupling is activated
+        indxBotm = GroundwaterSettings.indxBotmLayer; % index of bottom boundary layer after neglecting the saturated layers (from bottom to top)
```

Comment:
```suggestion
        % index of bottom layer, by default (no groundwater coupling) its layer
        % with index 1, since STEMMUS calculations start from bottom to top
        indxBotm = 1;
    else % Groundwater Coupling is activated
        % index of bottom boundary layer after neglecting the saturated layers
        % (from bottom to top)
        indxBotm = GroundwaterSettings.indxBotmLayer;
```

Reply:
Please avoid long inline comments as it increases the line length. Also, here some typos are fixed.

Reply:
> Please avoid long inline comments as it increases the line length. Also, here some typos are fixed.

Okay, thanks. I shrink the comments